build.go

```diff
@@ -51,8 +52,18 @@ func main() {
 	}
 }
 
+func runCmd(args ...string) error {
+	cmd := exec.Command(args[0], args[1:]...)
```

Comment:
I’m not sure if this hurts because it is only the `build.go` but it can result in an index out of bounds access if `len(args) < 1`.

Reply:
https://github.com/urfave/cli/pull/873/commits/1095838cca9d596e55cff88bcb35b67cf83bf4e4 ✔️